### PR TITLE
Add ScottoCMD Keyboard

### PR DESCRIPTION
**Pull request by joe-scotto**
## Description

Add the ScottoCMD keyboard and default keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/handwired/jscotto/scottocmd/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+// Draw to OLED
+bool oled_task_user() {
+    // Caps lock text
+    led_t led_state = host_keyboard_led_state();
+    oled_set_cursor(0,1);
+    oled_write_P(led_state.caps_lock ? PSTR(" Caps Lock "): PSTR(" ScottoCMD "), false);
+
+    //WPM text
+    oled_set_cursor(0, 9);
+    sprintf(wpm_str, "  %03d WPM  ", get_current_wpm()); // %03d defines digits to display
```
Comment (drashna) [inline keyboards/handwired/jscotto/scottocmd/keymaps/default/keymap.c:360]: https://docs.qmk.fm/#/squeezing_avr?id=oled-tweaks

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Also, ideally, the oled code should be moved to the kb level, that way, a default behavior is specified but can be overridden.  This especially benefits QMK Configurator users.

File: keyboards/handwired/jscotto/scottocmd/keymaps/default/keymap.c (hunk 1 above)
Comment (joe-scotto) [reply keyboards/handwired/jscotto/scottocmd/keymaps/default/keymap.c]: Thank you for that suggestion, updated my `keymap.c`.

**Comment by joe-scotto** [comment]
@drashna I put the OLED code in the default keymap because it relies on the layers there.

**Pull request merged**